tools/scylla-nodetool.cc

```diff
@@ -735,6 +739,8 @@ command-line arguments, the compaction will run on these tables.
 Fore more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/compact.html
 )",
                 {
+                    typed_option<>("skip-flush", "skip flushing tables before major compaction"),
```

Comment:
Here, too, we should use the positive sense.

Reply:
and maybe explain the implications a bit more

Reply:
ok, this is a switch, not a boolean, so can't reverse it

Reply:
right

Reply:
but it looks like we can use typed_option<bool> which defaults to `true`